docs-md/concepts/collections/overview.md

```diff
@@ -1,7 +1,35 @@
-Although an individual event represents vitally important information, in practice they are almost more useful when grouped together with related events. ksqlDB supports durable collections of events that can be replicated across multiple servers. This means that, similar to replicated Postgres, that if there is a recoverable failure of some of the servers, your events remain permanently stored. It is able to do this by storing data in Kafka topics.
+---
+layout: page
+title: Collections
```

Comment:
I was wondering why we introduced a new abstraction `Collections`. I guess it helps explain the common aspects of streams and tables in a uniform way?

Reply:
Yeah, that's right. It's just a piece of language to help say kind of thing streams and tables are, and to avoid saying both those words over and over when explaining something that applies to both. I don't think 'collections' should make its way into the code at all - just a tool for grouping concepts in our docs. What do you think?